README.md

```diff
@@ -1,40 +1,230 @@
-# database_based_on_diagram
+<a name="readme-top"></a>
 
-This a Blog app in development with rails and PostgreSQL.
 
-## Setup
+<div align="center">
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+</div>
```

Comment:
- [ ] As the previous reviewer suggested, please remove this `Microverse logo` in your README file. If you have our own logo, you can use it instead but If you don't, just don't put any logo here 😎 👍

Reply:
Done, thanks!